include/gatekeeper_lls.h

```diff
@@ -27,6 +27,14 @@
 #include "gatekeeper_acl.h"
 #include "gatekeeper_mailbox.h"
 
+#ifdef RTE_MACHINE_CPUFLAG_SSE4_2
+#include <rte_hash_crc.h>
+#define DEFAULT_HASH_FUNC       rte_hash_crc
+#else
+#include <rte_jhash.h>
+#define DEFAULT_HASH_FUNC       rte_jhash
+#endif
+
```

Comment:
Agreed, ``include/gatekeeper_main.h`` is a better place to hold this declaration.